File: two_pointers/container.py
height = [1,2]

def container(height):
  max_area = 0
  i = 0
  j = len(height) - 1
  while i < j:
    if height[j] > height[i]:
      product = height[i] * (j - i)
      i += 1
    else:
      product = height[j] * (j - i)
      j -= 1
    max_area = max(max_area, product)
  return max_area
  

result = container(height)
print(result)

File: two_pointers/container.js
let heights = [1, 2];

function container(heights) {
  let max_area = 0;
  let i = 0;
  let j = heights.length - 1;
  while (i < j) {
    let product = 1;
    if (heights[j] > heights[i]) {
      product = heights[i] * (j - i);
      i++;
    } else {
      product = heights[j] * (j - i);
      j--;
    }
    max_area = Math.max(max_area, product);
  }
  return max_area;
}

const result = container(heights);
console.log(result);

document.getElementById("result").textContent = "Resultado: " + result;
